co2eq/purchase/index.js

```diff
@@ -10,11 +10,27 @@ import {
   UNIT_ITEM,
   UNIT_LITER,
 } from '../../definitions';
-import { convertToEuro, getAvailableCurrencies } from '../../integrations/utils/currency/currency';
 import { getChecksum } from '../utils';
 import footprints from './footprints.yml';
 import consumerPriceIndex from './consumerpriceindices.yml';
 
+const exchangeRates2011 = require('./exchange_rates_2011.json');
+
+export function convertTo2011Euro(amount, currency) {
+  if (currency.toUpperCase() === 'EUR') return amount;
```

Comment:
You could also create a `const currencyUpperCase = currency.toUpperCase()` so you don't have to repeat it